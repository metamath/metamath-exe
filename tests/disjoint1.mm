$c var formula |- $.
$v x y $.
xf $f formula x $.
yf $f formula y $.
${
  $d x y $.
  ax-1 $a |- x y $.
$}
bad $p |- x x $=
  xf xf ax-1 $.
