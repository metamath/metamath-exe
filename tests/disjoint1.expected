MM> READ "disjoint1.mm"
Reading source file "disjoint1.mm"... 140 bytes
140 bytes were read into the source buffer.
The source has 9 statements; 1 are $a and 1 are $p.
No errors were found.  However, proofs were not checked.  Type VERIFY PROOF *
if you want to check them.
MM> Continuous scrolling is now in effect.
MM> 0 10%  20%  30%  40%  50%  60%  70%  80%  90% 100%
..................................................
?Error on line 10 of file "disjoint1.mm" at statement 9, label "bad", type
"$p":
  xf xf ax-1 $.
        ^^^^
There is a disjoint variable ($d) violation at proof step 3.  Assertion "ax-1"
requires that variables "x" and "y" be disjoint.  But "x" was substituted with
"x" and "y" was substituted with "x".  These substitutions have variable "x" in
common.

All proofs in the database were verified.
